### PR TITLE
Tag SQLite v0.2.1

### PR DESCRIPTION
**Pull request by Sean1708**

### CODE REVIEW

File: SQLite/versions/0.2.1/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.3-
```
Comment (tkelman) [inline SQLite/versions/0.2.1/requires]: remove the trailing `-`, this refers to prereleases of julia 0.3.

**Comment by Sean1708** [comment]
Is that everything?

**Comment by tkelman** [comment]
Yeah that should do it. Assuming @quinnj and other maintainers approve of tagging this commit?

**Comment by Sean1708** [comment]
[Yep](https://github.com/JuliaDB/SQLite.jl/issues/75).

**Comment by quinnj** [comment]
Thanks for the help @tkelman

**Pull request merged**